config_file

```diff
@@ -0,0 +1,21 @@
+port 1050;  # This is also a comment.
+
+path /static StaticHandler {
+  root static_src;
+}
+
+path / StaticHandler {
+  root static_src;
+}
+
+path /echo EchoHandler {}
+
+path /status StatusHandler {}
+
+path /proxy ProxyHandler {
+  host ucla.edu;
+  port 80;
+}
+
+# Default response handler if no handlers match.
+default BadRequestHandler {}
```

Comment:
I believe we accidentally added this file in a commit. I can just delete this because you guys already have config_file in /src